<<<<<<< HEAD
<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@848.v5a_383b_d14921">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@3731.ve4b_5b_857b_a_d3">
    <parentIds>
      <string>19</string>
    </parentIds>
    <id>20</id>
    <startId>3</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1251.vd4889a_b_0a_065">
      <startTime>1691876858736</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1251.vd4889a_b_0a_065">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@639.v6eca_cd8c04a_a_">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.interrupt(CpsFlowExecution.java:1166)</trace>
          <trace>org.jenkinsci.plugins.workflow.job.WorkflowRun$2.lambda$interrupt$0(WorkflowRun.java:398)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:67)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId plugin="workflow-api@1251.vd4889a_b_0a_065">
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>4d32ac82-8312-456a-abd6-23aa24f9a031</uuid>
          </wf.a.ErrorAction_-ErrorId>
        </suppressedExceptions>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <jenkins.model.CauseOfInterruption_-UserInterruption>
              <user>admin</user>
            </jenkins.model.CauseOfInterruption_-UserInterruption>
          </a>
        </causes>
        <actualInterruption>true</actualInterruption>
      </error>
    </wf.a.ErrorAction>
    <wf.a.ErrorAction plugin="workflow-api@1251.vd4889a_b_0a_065">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" reference="../../wf.a.ErrorAction/error"/>
    </wf.a.ErrorAction>
  </actions>
</Tag>
=======
version https://git-lfs.github.com/spec/v1
oid sha256:bcbfe3eebba2e7c18175da757858ef50877f574db2fecf163426303602142ee4
size 2828
>>>>>>> 01a95f3 (jenkins pipelines A-B)
